tests/test_tiff.py

```diff
@@ -66,8 +66,6 @@ def check_same(filename1, filename2, data_only=False):
     assert in1.size() == in2.size()
     assert in1.num_bands() == in2.num_bands()
     for i in range(in1.num_bands()):
-        if not data_only:
```

Comment:
Why did we remove this?

Reply:
This test is checking to see if we preserved the block size of the tif through read/write but that isn't something our software is programmed to do.  We could add it is a feature but I don't think it is necessary.